README.md

```diff
@@ -125,6 +125,29 @@ dashboardTemplates:
 
 Note that you can define more than one dashboard under the `dashboardTemplates` field. 
 
+#### Entity TTL
+
+Entities are deleted by default 8 days after the last time they were indexed or sent out a heartbeat. 
+If this doesn't suit your needs you may also specify your desired behavior under the TTL section of the main `definition.yml` file:  
+
+```yaml
+ttl:
+  # The strategy to follow for the entity deletion. Defaults to AUTOMATIC
+  deletionStrategy: AUTOMATIC
+  # The frequency to delete the entity types. Defaults to EIGHT_DAYS
+  deletionFrequency: EIGHT_DAYS
+```
+
+`deletionStrategy` accepts the values `AUTOMATIC` (default) and `MANUAL`. 
+
+If `AUTOMATIC` is used the entity will be automatically deleted after a certain amount of time, defined by the `deletionFrequency`: 
```

Comment:
Some notes:

* I'd use `expirationPolicy` and `expirationTime`.  Frequency is somewhat confusing for me (I read it as "they get deleted every X days"). Maybe @jfjoly has an opinion here?

* Reword: "will be automatically deleted **if we don't receive any telemetry** for the time defined by `expirationTime`".  Otherwise it reads as if we will delete any entity regardless.